app/src/main/java/org/simple/clinic/settings/SettingsRepositoryImpl.kt

```diff
@@ -0,0 +1,19 @@
+package org.simple.clinic.settings
+
+import io.reactivex.Completable
+import io.reactivex.Single
+
+class SettingsRepositoryImpl : SettingsRepository {
```

Comment:
This class is fine for now because the screen is unreachable from the UI. The final integration will happen in the next PR.